### PR TITLE
Yukon Age Amount

### LINKED ISSUE #293: Yukon Age Amount
reference:
https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf

### PR DESCRIPTION
**Pull request by RuoqiTan**
fixes #293

### CODE REVIEW

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: The eligible age for getting the age amount of Yukon Personal Tax Credits Return.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:1]:
```suggestion
description: Yukon provides the age amount credit for filers above this age.
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+values:
+  2022-01-01: 65
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:3]:
```suggestion
  2023-01-01: 65
```

Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:3]: change to 2023 in other parameters as well

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_net_income.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_net_income.yaml:1]: Can delete this as this will be mathematically derived

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml:1]: make this a "marginal rate" parameter with the rate (15% over 42_335) -> adjust variable after

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:1]: make this a "marginal rate" parameter with the base_amount (15% over 42_335) -> adjust variable after

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml (hunk 1 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:1]: Other descriptions similar: Yukon does xyz under the age amount credit.

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximun_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximun_amount.yaml:1]: delete duplicate file

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+description: Yukon base amount of one's net income for the age amount of Yukon Personal Tax Credits Return.
+values:
+  2022-01-01: 42_335
+metadata:
+  unit: currency-CAD
+  label: Yukon base amount
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml:6]:
```suggestion
  label: Yukon age amount base
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+metadata:
+  unit: year
+  label: Yukon age amount eligible age
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:8]: need to add Yukon income tax act as a source

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+description: Yukon Maximum amount that one can get for the age amount of Yukon Personal Tax Credits Return.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml:1]:
```suggestion
description: Yukon Maximum amount that one can get for the age amount of Yukon Personal Tax Credits Return.
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_net_income.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+description: Yukon maximum net income for one to get the age amount of Yukon Personal Tax Credits Return.
+values:
+  2022-01-01: 98_309
+metadata:
+  unit: currency-CAD
+  label: Yukon age amount maximum net income
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_net_income.yaml:6]:
```suggestion
  label: Yukon age amount income threshold
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_net_income.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_net_income.yaml:1]: rename to `income_threshold.yaml`

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximun_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximun_amount.yaml:1]: duplicate yaml?

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+description: Rate of the age amount of Yukon Personal Tax Credits Return.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:1]: adj. description

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml (hunk 4 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:8]: all yaml

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+description: Yukon provides this rate under the age amount credit.
+values:
+  2023-01-01: 0.15
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:3]:
```suggestion
  2022-01-01: 0.15
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+metadata:
+  unit: /1
+  label: Yukon age amount applicable rate
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
+  - title: Income Tax Act
+    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:11]: add 2022 reference https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-22e.pdf

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml [hunk 10]
```diff
@@ -0,0 +1,11 @@
+description: Yukon provides this maximum amount that one can get under the age amount credit.
+values:
+  2023-01-01: 8_396
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml:3]:
```suggestion
  2022-01-01: 7_898
  2023-01-01: 8_396
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:1]: make scale parameter same as in: https://github.com/PolicyEngine/policyengine-canada/pull/354/files#diff-37fc830fd6bb7e4f47737cac2188fe57211a4976c8921319c53910161937db11

**Comment by RuoqiTan** [comment]
How to use the marginal rate? It can't be calculated with (income * rate).

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+values:
+  2022-01-01: 39_826
+  2023-01-01: 42_335
+metadata:
+  unit: currency-CAD
+  label: Yukon age amount base
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
+  - title: Income Tax Act
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml:11]: all yaml

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml [hunk 12]
```diff
@@ -0,0 +1,14 @@
+description: Yukon provides this maximum amount that one can get under the age amount credit.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml:1]:
```suggestion
description: Yukon provides this maximum amount under the age amount credit.
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/base_amount.yaml:1]: delete file

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 13]
```diff
@@ -0,0 +1,27 @@
+description: Yukon provides this rate under the age amount credit.
+brackets:
+  - threshold:
+      2022-01-01: 0
+    rate:
+      2022-01-01: 0
+  - threshold:
+      2022-01-01: 39_826
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:8]:
```suggestion
      2022-01-01: 39_826
      2023-01-01: 42_335
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+    rate:
+      2022-01-01: 0.15
+  - threshold:
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:11]: delete line 11 - 14

File: policyengine_canada/tests/gov/provinces/yt/tax/income/credits/yt_age_amount.yaml [hunk 15]
```diff
@@ -0,0 +1,35 @@
+- name: Not eligible for age
```
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/yt/tax/income/credits/yt_age_amount.yaml:1]:
```suggestion
- name: Not eligible for age amounr
```

File: policyengine_canada/tests/gov/provinces/yt/tax/income/credits/yt_age_amount.yaml [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+  period: 2023
+  input:
+    province_code: YT
+    individual_net_income: 10_000
+    age: 60
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_canada/tests/gov/provinces/yt/tax/income/credits/yt_age_amount.yaml:6]: test edge cases -> off by 1

File: policyengine_canada/tests/gov/provinces/yt/tax/income/credits/yt_age_amount.yaml [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+  output:
+    yt_age_amount: 0
+
+- name: Not eligible for net income
```
Comment (PavelMakarchuk) [inline policyengine_canada/tests/gov/provinces/yt/tax/income/credits/yt_age_amount.yaml:10]:
```suggestion
- name: Not eligible through net income
```

File: policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_age_amount.py [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+    defined_for = ProvinceCode.YT
+
+    def formula(person, period, parameters):
+        income = person("individual_net_income", period)
+        age = person("age", period)
+        p = parameters(period).gov.provinces.yt.tax.income.credits.age_amount
+        # reduction = (
+        #     (income > p.base_amount) * (income - p.base_amount) * p.rate
+        # )
+        reduction = income * p.rate
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_age_amount.py:18]:
```suggestion
        reduction = p.rate.calc(income)
```

File: policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_age_amount.py [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+        reduced_amount = max_(p.maximum_amount - reduction, 0)
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_age_amount.py:19]: add line eligible = age >= p.age

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 20]
```diff
@@ -0,0 +1,14 @@
+description: Yukon provides the age amount credit for filers above this age.
+
+values:
+  2023-01-01: 65
+metadata:
+  unit: year
+  label: Yukon age amount eligible age
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:10]:
```suggestion
    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf#page=1
```

Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:10]: and other yaml and references

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/income_threshold.yaml
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/income_threshold.yaml:1]: dont need this as this is mathematically derived

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 21]
```diff
@@ -0,0 +1,24 @@
+description: Yukon provides this rate under the age amounhttps://ruoqitan-didactic-engine-gvx7v4694ppcp47x.github.dev/t credit.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:1]:
```suggestion
description: Yukon reduces the age amount credit at this rate.
```

File: policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_age_amount.py [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+    defined_for = ProvinceCode.YT
+
+    def formula(person, period, parameters):
+        income = person("individual_net_income", period)
+        age = person("age", period)
+        p = parameters(period).gov.provinces.yt.tax.income.credits.age_amount
+        reduction = p.rate.calc(income)
+        reduced_amount = max_(p.maximum_amount - reduction, 0)
+        age_eligible = age >= p.age
+        income_eligible = income < p.income_threshold
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_age_amount.py:18]:
```suggestion
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 23]
```diff
@@ -0,0 +1,15 @@
+description: Yukon qualifies filer for the age amount credit at or above this age threshold.
+
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:2]:
```suggestion
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 24]
```diff
@@ -0,0 +1,15 @@
+description: Yukon qualifies filer for the age amount credit at or above this age threshold.
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:1]:
```suggestion
description: Yukon qualifies filers for the age amount credit at or above this age threshold.
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 25]
```diff
@@ -0,0 +1,15 @@
...
+values:
+  2023-01-01: 65
+metadata:
+  unit: year
+  label: Yukon age amount eligible age
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf#page=1
+  - title: Income Tax Act -- Page 8
+    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:13]:
```suggestion
    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf#page=8
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 26]
```diff
@@ -0,0 +1,15 @@
...
+
+values:
+  2023-01-01: 65
+metadata:
+  unit: year
+  label: Yukon age amount eligible age
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf#page=1
+  - title: Income Tax Act -- Page 8
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:12]: all yaml

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml (hunk 25 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:13]: all yaml

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 27]
```diff
@@ -0,0 +1,13 @@
+description: Yukon qualifies filers for the age amount credit at or above this age threshold.
+values:
+  2023-01-01: 65
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:3]:
```suggestion
  2022-01-01: 65
```

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml [hunk 28]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  unit: currency-CAD
+  label: Yukon age amount maximum amount
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
+  - title: Yukon Income Tax Act -- Age Credit
+    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf#page=8
+  - title: Worksheet for the Yukon 2022 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-22e.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml:14]: need page here

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml [hunk 29]
```diff
@@ -0,0 +1,14 @@
+description: Yukon provides this maximum amount under the age amount credit.
+values:
+  2022-01-01: 7_898
+  2023-01-01: 8_396
+metadata:
+  unit: currency-CAD
+  label: Yukon age amount maximum amount
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml:10]: need page here

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml [hunk 30]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  unit: year
+  label: Yukon age amount eligible age
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf#page=1
+  - title: Yukon Income Tax Act -- Age Credit
+    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf#page=8
+  - title: Worksheet for the Yukon 2022 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-22e.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/age.yaml:13]: need page here

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 31]
```diff
@@ -0,0 +1,24 @@
...
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  label: Yukon age amount applicable rate
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
+  - title: Yukon Income Tax Act -- Age Credit
+    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf#page=8
+  - title: Worksheet for the Yukon 2022 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-22e.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:24]: need page here

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml [hunk 32]
```diff
@@ -0,0 +1,24 @@
...
+      2022-01-01: 0.15
+
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  label: Yukon age amount applicable rate
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/rate.yaml:20]: need page here

File: policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml [hunk 33]
```diff
@@ -0,0 +1,14 @@
...
+  2022-01-01: 7_898
+  2023-01-01: 8_396
+metadata:
+  unit: currency-CAD
+  label: Yukon age amount maximum amount
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf#page=1
+  - title: Yukon Income Tax Act -- Age Credit
+    href: https://laws.yukon.ca/cms/images/LEGISLATION/PRINCIPAL/2002/2002-0118/2002-0118.pdf#page=8
```
Comment (PavelMakarchuk) [inline policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/age_amount/maximum_amount.yaml:12]: does this page reference the age credit? Not sure if I can see it

**Comment by PavelMakarchuk** [comment]
Duplicate #489
